### PR TITLE
Include Freemius into Events Calendar admin

### PR DESCRIPTION
**Pull request by bordoni**
_Ref:_ [C#124015](http://central.tri.be/issues/124015)
---

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | The method parameter $message is never used | [Freemius.php:83](https://github.com/moderntribe/the-events-calendar/blob/a6bf23f72ac4f68630ab6dbc800d13598a8d4116/src/Tribe/Integrations/Freemius.php#L83)

> via jenkins/codesniffer-phpcs

File: src/Tribe/Integrations/Freemius.php [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+	public function filter_connect_message_on_update(
+		$message,
+		$user_first_name,
+		$product_title,
+		$user_login,
+		$site_link,
+		$freemius_link
+	) {
+		$plugin_name = 'The Events Calendar';
+		$title = '<h3>' . sprintf( __( 'We hope you love %1$s', 'the-events-calendar' ), $plugin_name ) . '</h3>';
```
Comment (sc0ttkclark) [inline src/Tribe/Integrations/Freemius.php]:
```suggestion
		$title = '<h3>' . sprintf( esc_html__( 'We hope you love %1$s', 'the-events-calendar' ), $plugin_name ) . '</h3>';
```

File: src/Tribe/Integrations/Freemius.php [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+		$html = '';
+
+		$html .= '<p>';
+		$html .= sprintf(
+			__( 'Hi, %1$s! This is an invitation to help %2$s community. If you opt-in, some data about your usage of %2$s will be shared with our teams (so they can work their butts off to improve). We will also share some helpful info on events management. WordPress, and our products from time to time.', 'the-events-calendar' ),
```
Comment (sc0ttkclark) [inline src/Tribe/Integrations/Freemius.php]:
```suggestion
			esc_html__( 'Hi, %1$s! This is an invitation to help %2$s community. If you opt-in, some data about your usage of %2$s will be shared with our teams (so they can work their butts off to improve). We will also share some helpful info on events management. WordPress, and our products from time to time.', 'the-events-calendar' ),
```

File: src/Tribe/Integrations/Freemius.php [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+			$user_first_name,
+			$plugin_name
+		);
+		$html .= '</p>';
+
+		$html .= '<p>';
+		$html .= sprintf(
+			__( 'And if you skip this, that\'s okay! %1$s will still work just fine.', 'the-events-calendar' ),
```
Comment (sc0ttkclark) [inline src/Tribe/Integrations/Freemius.php]:
```suggestion
			esc_html__( 'And if you skip this, that\'s okay! %1$s will still work just fine.', 'the-events-calendar' ),
```

File: src/Tribe/Integrations/Freemius.php [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+			$plugin_name
+		);
+		$html .= '</p>';
+
+		// Powered by HTML
+		$html .= '<div class="tribe-powered-by-freemius">' . __( 'Powered by', 'the-events-calendar' ) . '</div>';
```
Comment (sc0ttkclark) [inline src/Tribe/Integrations/Freemius.php]:
```suggestion
		$html .= '<div class="tribe-powered-by-freemius">' . esc_html__( 'Powered by', 'the-events-calendar' ) . '</div>';
```

File: src/Tribe/Integrations/Manager.php [hunk 5]
```diff
@@ -33,12 +33,38 @@ public static function instance() {
...
+		$this->load_freemius();
 		$this->load_acf_integration();
 		$this->load_twenty_seventeen_integration();
 		$this->load_wpml_integration();
 		$this->load_X_theme_integration();
 	}
 
+	/**
+	 * Loads our Events Freemius integration
+	 *
```
Comment (sc0ttkclark) [inline src/Tribe/Integrations/Manager.php]:
```suggestion
	 *
	 * @since TBD
	 *
```

**Pull request merged**